### ISSUE TITLE
Cannot load main class (GNU/linux and mac)

### ISSUE THREAD

**Issue by Ganjalf-Amsterdoom** [SKCraft/Launcher#37]
Hello,

since I moved to forge 1.7.10 I cannot load (forge) on GNU/linux and macs.

Should I post you my manifest and/or my version file?

On windows modpacks start properly.

**Comment by Ganjalf-Amsterdoom** [comment]
Could it be related to the launchwrapper library? An earlier version of forge 1.7.10 with a different launchwrapper lib starts normally on linux and mac.

**Comment by sk89q** [comment]
How are you running the .jar on Linux/Mac?

**Comment by Ganjalf-Amsterdoom** [comment]
On gnu/linux java -jar launcher.jar, on mac by click-open.
I forgot to mention I'm running without the bootstrapper and I edited the source to create a ~/.mylauncher folder but I also recompiled clean source (only editing resources-lang) and its the same result.
Let me show you how I compile the modpack, perhaphs is that?

/opt/java/64/jre1.7.0_71/bin/java -cp ~/Launcher/launcher-vecchio/target/Launcher.jar com.skcraft.launcher.builder.PackageBuilder --version "3.7" --manifest-dest "dist/manifest.json" --objects-dest "dist/objects" --files "src" --config "modpack.json" --objects-url "http://mysitecensored/ModPacks/mycollection/dist/objects" --version-file "version.json" --libs-url "http://mysitecensored/ModPacks/mycollection/libraries"
